src/app/clusters/fan-control-server/fan-control-server.cpp

```diff
@@ -85,6 +85,10 @@ namespace {
 // Indicates if the write operation is from the cluster server itself
 bool gWriteFromClusterLogic = false;
 
+// Avoid circular callback calls when adjusting SpeedSetting and PercentSetting together.
+bool gSpeedWriteInProgress   = false;
```

Comment:
```suggestion
static bool gSpeedWriteInProgress   = false;
```

Reply:
Moving both inside the anonymous namespace above would work as well

Reply:
They're inside the anonymous namespace. It starts on line 83